solidity/dashboard/src/components/AuthorizeContracts.jsx

```diff
@@ -153,14 +140,39 @@ const AuthorizeContractItem = ({
             )}
           </div>
         ) : (
-          <SubmitButton
-            onSubmitAction={onAuthorize}
-            className="btn btn-secondary btn-sm"
-            style={{ marginLeft: "auto" }}
-            successCallback={onSuccess}
+          <Tooltip
+            shouldShowTooltip={
+              contractName === "Keep Random Beacon Operator Contract"
```

Comment:
Would be best to save this `contractName` as constants somewhere. Not only `Keep Random Beacon Operator Contract` but i see there are also `TBTCSystem` and `BondedECDSAKeepFactory` that are used in multiple places.

Those name are different than the names in the `constants.js` file (e.g in `constants.js` there is `bondedEcdsaKeepFactoryContract` instead of `BondedECDSAKeepFactory`) which makes it confusing 🤔 Maybe we should create separate constants and call it `CONTRACT_LABEL` or something? Thoughts?